src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -397,6 +399,10 @@ def create_or_update_test_with_config(
             new_body["keyvaultReferenceIdentityType"] = IdentityType.SystemAssigned
             new_body.pop("keyvaultReferenceIdentityId")
     subnet_id = subnet_id or yaml_test_body.get("subnetId")
+    if disable_public_ip:
```

Comment:
are we giving preference to what is being sent from the cli param as compared to yaml here? if the user sets cli param as "false" this if block would not resolve to true? specially if the user initially just set it as true from yaml , but then they use cli param to try to enable public ip again.

Reply:
Good catch Mouly, I will update the condition to check if disable_public_ip is not None. This would ensure considering param only if a true or false value is provided. Yes the preference would be value provided in param over value provided via config, in case both are present.